amethyst_assets/src/loader.rs

```diff
@@ -25,6 +26,7 @@ impl Loader {
     {
         Loader {
             directory: Arc::new(Directory::new(directory)),
+            hot_reload: true, // TODO: allow disabling
```

Comment:
Should we really default to true here? Even for the first iteration?

Reply:
No, I have to add configs to this; the param is just `true` to try it out.